src/layouts/index.js

```diff
@@ -6,6 +6,7 @@ import '../components/css/custom.scss';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import Header from './Header';
 import '../assets/images/512.png';
+import CategoryList from 'theme/components/CategoryList';
```

Comment:
One point to note here is, when we are writing a new theme with some new set of component, CategoryList component should be exported for sure and that component can be built using data and api functions passed in as props to the `CategoryList` component. This is just an example using `CategoryList` page alone. In the same way, some basic component should be exported from the theme for sure(like productpage, cartpage etc)